pkg/ec2metadatawrapper/ec2metadatawrapper.go

```diff
@@ -0,0 +1,64 @@
+/*
+  Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  A copy of the License is located at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  or in the "license" file accompanying this file. This file is distributed
+  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+  express or implied. See the License for the specific language governing
+  permissions and limitations under the License.
+*/
+
+// Package ec2metadatawrapper is used to retrieve data from EC2 IMDS
+package ec2metadatawrapper
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/ec2metadata"
+	"github.com/aws/aws-sdk-go/aws/session"
+)
+
+const (
+	metadataRetries = 5
+)
+
+// HttpClient is used to help with testing
+type HttpClient interface {
+	GetInstanceIdentityDocument() (ec2metadata.EC2InstanceIdentityDocument, error)
+	Region() (string, error)
+}
+
+// EC2MetadataClient to used to obtain a subset of information from EC2 IMDS
+type EC2MetadataClient interface {
+	GetInstanceIdentityDocument() (ec2metadata.EC2InstanceIdentityDocument, error)
```

Comment:
Is this method being used anywhere?

Reply:
No, I used https://github.com/aws/amazon-vpc-cni-k8s/blob/master/pkg/ec2metadatawrapper/ec2metadatawrapper.go that's why the method is there.
I will remove that part of the code.